SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+  %The proposed timeline for release of PVIs is in the Project review process and may be revised.
+  This document is still being drafted.
+  Release of associated source catalogs with the PVIs is not in the baseline.
+  The proposed timeline for the release of PVIs and release of associated source catalogs are in the Project review process. After that, this document will be revised accordingly.
+\end{warning}
+
+%To support early science by the community,
+Rubin Observatory seeks to maximize opportunities for early science with LSST data by the community.
+At the start of the LSST 10-year survey, if the processing pipelines are sufficiently stable, nightly Processed Visit Images (PVIs) and accompanying source catalogs will be released on a best-effort basis.
+%At the start of the LSST 10-year survey, if the processing pipelines are sufficiently stable, nightly Processed Visit Images (PVIs) will be released on a best-effort basis.
+Release is subject to all embargo requirements, with caveats and preliminary calibration information.
+%as determined by the Project and reviewed by [the SAC?, Ops Data Release Board?]
+%These would be labeled as Preliminary Data, with
+%There will be caveats and preliminary information about calibration.
+% and no Project support for use until DP2.
+%After the Data Preview 2 release
+%Within 6 months following the start of LSST, PVIs will be released within the standard time window for steady-state LSST operations (see \citedsp{RTN-011}).
+Within 6 months following the start of LSST, PVIs and source catalogs will be released within the standard time window for steady-state LSST operations (see \citedsp{RTN-011}).
+
+\section{Commissioning Era Information Sharing Policies}
+\label{policy}
+
+\subsection{Summary}
+
+%Project team members must follow policies for embargoed data products described in the Vera C. Rubin Observatory Data Policy \citedsp{RDO-013} and Project Publication Policy \citedsp{LPM-162}.
+%, and Rubin Observatory Data Security Standards Implementation (DMTN-199).
+All pixel images and representations of pixel images of any size field of view, including individual visit images, coadd images, and difference images based on LSSTCam commissioning on-sky observations are embargoed for a period of at least 30 days, with the exception of alert postage stamp images.
+
+During the period from installation of LSSTCam on the telescope to the release of Data Preview 2, all communications (including informal discussion) regarding proprietary engineering and on-sky data with LSSTCam are internal to Rubin Project by default.
+Outward-facing communications, including all on-sky LSSTCam images, are reviewed by SIT-Com leadership, the Rubin Celebrations Organizing Committee (RCOC) and its relevant subgroups, and the Communications team, and are approved for release by the Project Director or designated alternate.
+
+Example outward-facing communications:
```

Comment:
Not exactly fully outward facing but the MREFC monthly status report to NSF

Reply:
I had thought that communication with funding agencies is included as being "internal".